draft-ietf-taps-impl.md

```diff
@@ -942,7 +944,7 @@ Connectedness: Connected
 Data Unit: Byte-stream
 
 the Transport Services API mappings for MPTCP are identical to TCP. MPTCP adds support for multipath properties,
-such as "Multipath Transport" and "Policy for using Multipath Transports".
+such as `multipath` and `multipath-policy`.
```

Comment:
For sake of consistency: shouldn't we better use `multipathPolicy` here?

Reply:
I think the API is multipath-policy

Reply:
We'll have a separate issue to turn all of them into camel case in the interface draft (another example: `advertises-altaddr`).